modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -29,6 +29,9 @@ class OrtResultAnalyzer : ManualAnalyzer() {
         private val LOGGER = LoggerFactory.getLogger(OrtResultAnalyzer::class.java)
     }
 
+    private val omitExcludedArtifactsProperty = "omit.excluded"
+    private var omitExcludedArtifacts = false
```

Comment:
Didn't we also agree to set this default to `true`, or did I misunderstand something?

Reply:
Also, if `configure()` is guranteed to *always* be called, and always *before* `createArtifactList()`, then we should make this a `private lateinit val omitExcludedArtifacts: Boolean`.

Reply:
I reverse the logic, so that you have to configure this, if you do not want the scope reduced